tiledb/sm/cpp_api/query.h

```diff
@@ -382,8 +382,9 @@ class Query {
       auto attr_name = b_it.first;
       auto size_pair = b_it.second;
       auto var =
-          (attr_name != TILEDB_COORDS &&
-           schema_.attribute(attr_name).cell_val_num() == TILEDB_VAR_NUM);
+          ((attr_name != TILEDB_COORDS) &&
+              ((schema_.has_attribute(attr_name) && schema_.attribute(attr_name).cell_val_num() == TILEDB_VAR_NUM) ||
+              (schema_.domain().has_dimension(attr_name) && schema_.domain().dimension(attr_name).cell_val_num() == TILEDB_VAR_NUM)));
```

Comment:
`schema_.var_size(name)` would work for both dimensions and attributes.

Reply:
`tiledb::ArraySchema` (cppapi) doesn't have that method. Should I add it?

Reply:
Yes please, I thought we had it there.